docs/development/maintainers.md

```diff
@@ -108,3 +108,18 @@ final number. Follow the relevant steps from {ref}`release-instructions`.
 
 Cherry pick the corresponding documentation commits to the `stable` branch. Use
 `[skip ci]` in the commit message.
+
+### Upgrading pyodide to a new version of CPython
+
+For example: `v3.11.1` -> `v3.11.2`
+
+1. Download the **Gzipped source tarball** at https://www.python.org/downloads/release/python-3112 into `downloads/`
+2. `shasum -a 256 downloads/Python-3.11.2.tgz > cpython/checksums`
+3. `git grep --name-only "3.11.1" ` # All these files will need to be updated.
+4. After updating the Python version in `Dockerfile`, create a new Docker image.
+   - A maintainer must click `Run workflow` on https://github.com/pyodide/pyodide/actions/workflows/docker_image.yml
+5. That workflow will build and upload a new Docker image to https://hub.docker.com/r/pyodide/pyodide-env/tags
+6. Modify the image name in `.circleci/config.yml` to match the name Docker Hub.
+   - `image: pyodide/pyodide-env:20230301-chromelatest-firefoxlatest`
```

Comment:
@rth can we get it to correctly version the chrome and firefox versions? It's kind of dumb for it to say latest...

Reply:
This was done and pushed.